### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by alayek**
# DO NOT MERGE

This is still WIP. Each of you can push changes on how to use your stack. Then we merge

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,194 @@
...
+	* [learnyounode](https://github.com/workshopper/learnyounode)
+	* [how-to-npm](https://github.com/npm/how-to-npm)
+	* [learnyoumongo](https://github.com/evanlucas/learnyoumongo)
+	* [expressworks](https://github.com/azat-co/expressworks)
+
+In vagrant configuration, the port 3000 is mapped with guest port 9200. Thus, if you start an express app on port 3000, you can access it via the address [http://localhost:9200](http://localhost:9200)
+
+We have also included a sample [Todo app](https://github.com/tastejs/todomvc/tree/master/examples/socketstream/) with Socket Stream, built by the awesome guys at [TodoMVC](https://github.com/tastejs/todomvc). To run it, do the following :
+
+```
```
Comment (alayek) [inline README.md]: Add `bash` syntax highlighting. No worries I will add it in next commit.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,225 @@
...
+
+Now open [http://localhost:9200](http://localhost:9200) in your browser and voila! The todo app is working!
+
+### PHP
+
+### Python
+
+### Ruby
+
+In Ruby box we have provided the following packages :
```
Comment (ghost) [inline README.md]: ~~In Ruby box~~ In **the** Ruby box

File: README.md [hunk 3]
```diff
@@ -2,18 +2,225 @@
...
+* Gradle 2
+* Tomcat 8
+
+Additionally, Postgres database has been setup.
+
+Tomcat server is set to auto-start on port 8080. Now in this box, the port 8080 is mapped with Host, i.e your machine's port 9100. Thus, run anything on the tomcat server and it would be accessible on your machine in the address [http://localhost:9100](http://localhost:9100)
+
+### Node
+
+In Node box we have provided the following packages :
```
Comment (ghost) [inline README.md]: ~~In Node box~~ In **the** Node box

File: README.md [hunk 4]
```diff
@@ -2,18 +2,225 @@
...
+* [Yeoman](http://yeoman.io/) CLI
+* [Express](http://expressjs.com/)
+* Essential Nodeschool Workshopper modules
+	* [git-it](https://github.com/jlord/git-it)
+	* [learnyounode](https://github.com/workshopper/learnyounode)
+	* [how-to-npm](https://github.com/npm/how-to-npm)
+	* [learnyoumongo](https://github.com/evanlucas/learnyoumongo)
+	* [expressworks](https://github.com/azat-co/expressworks)
+
+In vagrant configuration, the port 3000 is mapped with guest port 9200. Thus, if you start an express app on port 3000, you can access it via the address [http://localhost:9200](http://localhost:9200)
```
Comment (ghost) [inline README.md]: ~~In vagrant configuration,~~ In **our** vagrant configuration,

**Comment by byteknacker** [comment]
@atjonathan did you make the changes to the files? Do you have access to change them?

File: python/README.md [hunk 5]
```diff
@@ -1,5 +1,158 @@
...
+```
+
+You should see all tests PASSED and no errors printed.
+
+# Contributing Guidelines
+
+## For people who wish to write new code
+
+1. Fork the main GitHub repo of Zeus
+2. Read the the CONTRIBUTING.md and Wiki pages to get yourself up to speed
```
Comment (ghost) [inline python/README.md]: ~~Read the the CONTRIBUTING.md~~ Read **our `[Contributing Docs](https://github.com/alayek/zeus/blob/master/.github/CONTRIBUTING.md)`**

File: python/README.md [hunk 6]
```diff
@@ -1,5 +1,158 @@
...
+4. Submit Pull Request from your fork's master branch
+
+## For people who wish to review existing code
+
+1. Pull in the changes from main `master` branch or any pull requests and test them locally on your machine.
+2. Follow the steps under the section "Test Your Setup". Use multiple testing methods.
+
+Do not be alarmed by any red-colored text during provisioning. Unless it clearly says _error_ no need to panic!
+
+# Acknowledgement
```
Comment (ghost) [inline python/README.md]: ~~Acknowledgement~~ Acknowledgement**s**

File: README.md [hunk 7]
```diff
@@ -2,18 +2,225 @@
...
+
+> Also, most such projects are built to complement an existing project of the authors who created the project. They most likely won't work on your personal Windows 7 laptop. They are not usually active, and defintely not meant for mass consumption.
+
+> We encountered an interesting bug when we were following a popular repo's vagrant provisioning - it has issues on some Windows machine. Upon further inspection, we found that there was a step to copy `~/.bashrc` from the cloned repo into the vagrant box. Guess what, if the cloning git client is not set up to handle line terminators properly, the vagrant box would only see Windows style line terminators and the whole `~/.bashrc` sourcing would fail with lot of error texts.
+
+> We wanted to get rid of all these, so decided to have our own boxes. But the good practices of all these repositories, still live on. We incorporate them whenever possible, with proper credit to original author.
+
+### Why PostgreSQL, and no MySQL?
+
+> Because our campers prefer heroku for deployment. Heroku is simpler than AWS or gcloud or Google App Engine. And by deafult, PostgreSQL is supported on Heroku.
```
Comment (ghost) [inline README.md]: ~~And by deafult,~~ And by **default,**

**Comment by ghost** [comment]
@byteknacker, I've requested permission for access by @alayek, should be fine! I'll add a commit shortly with the fixes.

**Comment by byteknacker** [comment]
@atjonathan many thanks for your time!

File: README.md [hunk 8]
```diff
@@ -2,18 +2,257 @@
...
+    * php5-mcrypt 
+    * php5-sqlite 
+    * php5-apcu 
+    * php5-cli
+    * php5-gd
+  * PostgreSQL server 9.3 (MySQL or MariaDB support may come later)
+  * PostgreSQL client 9.3
+  * phpunit 3.7
+
+The typical workflow imagined for this PHP box is the following :
```
Comment (ghost) [inline README.md]: ~~The typical workflow imagined for this PHP box is the following :~~ The typical workflow that was imagined for this PHP box:

File: README.md [hunk 9]
```diff
@@ -2,18 +2,257 @@
...
+    - From the guest : you will have to configure your git repository, but the Heroku toolbelt is installed.
+  5. Repeat from 2
+  6. Celebrate.
+
+#### Defaults
+
+  * The default IP adress of the guest is `http://192.168.56.101`.
+  * You can access the PostgreSQL server with user `vagrant`, password `vagrant`. An empty database named `my_app` has already been created.
+
+For complete informations, refer to the [PHP readme](php/README.md).
```
Comment (ghost) [inline README.md]: ~~complete informations~~ more information

**Pull request merged**